tests/transports-integrations/tests/utils/models.py

```
@@ -0,0 +1,73 @@
+"""
+Model configurations for each integration.
+
+This file now acts as a compatibility layer and convenience wrapper
+around the new configuration system in config.yml and config_loader.py.
+
+All model data is now centralized in config.yml for easier maintenance.
+"""
+
+from typing import Dict, List
```

Ruff (0.11.9)

`typing.Dict` is deprecated, use `dict` instead

(UP035)